### PR TITLE
Fix builds to find one YAML file 

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: readthedocs_build/config/config.py [hunk 1]
```diff
@@ -338,15 +338,21 @@ def set_output_base(self, directory):
...
     given path. That is usually the root of the project.
 
     The config will be validated.
+
+    `only` means not to search for config files, but use the one passed in
     """
 
-    config_files = list(find_all(path, CONFIG_FILENAMES))
+    if only:
+        config_files = [path]
```
Comment (safwanrahman) [inline readthedocs_build/config/config.py]: I think the `path` contain the repo path only. not the absolute path of `readthedocs.yml`

Comment (ericholscher) [reply readthedocs_build/config/config.py]: Yea, this will be fixed on the RTD side.

**Comment by agjohnson** [comment]
Instead of special casing `only`, why don't we just take the top-most config? Merging configs seems like unwieldy magic. We shouldn't be implementing features here and then altering those implementations on RTD.

I'd be :+1: on just using the top most file, I don't see a use case for merging multiple configs. If we need merging, we should have an `include` directive.

**Comment by ericholscher** [comment]
@agjohnson @safwanrahman New approach here. Think it makes more sense, and doesn't require .org changes.

**Pull request merged**